cli/package.json

```diff
@@ -0,0 +1,58 @@
+{
+  "name": "local-home-testing-cli",
+  "version": "0.0.1",
+  "description": "A command-line interface for interacting with Local Home SDK stubs.",
+  "private": true,
+  "author": "Google LLC.",
+  "license": "Apache-2.0",
+  "main": "./build/src/index.js",
+  "scripts": {
+    "compile": "tsc",
+    "test": "ava",
+    "clean": "gts clean",
+    "cli": "node ./build/src/index.js",
+    "lint": "eslint './src/**/*.ts' './test/**/*.ts'"
```

Comment:
should that be `gts check` ? (i.e: what result of `gts init`)

Reply:
Went through both projects and removed most the `eslint` dependencies.

As a side note, some `eslint` dependencies seem to be a bit outdated in `gts`.  In particular, I had to include the latest version of `@typescript-eslint/eslint-plugin` since it was seemingly required to recognize the Local Home SDK types through triple-slash directives.